pkgs/nix-directory.nix

```diff
@@ -56,9 +52,12 @@ stdenv.mkDerivation {
       cp --archive "$i" "${buildRootDirectory}$i"
     done
 
-    USER=${config.user.userName} ${prootCommand} "$PKG_NIX/bin/nix-store" --init
-    USER=${config.user.userName} ${prootCommand} "$PKG_NIX/bin/nix-store" --load-db < .reginfo
-    USER=${config.user.userName} ${prootCommand} "$PKG_NIX/bin/nix-store" --load-db < ${prootTermuxClosure}/registration
+    ls -l /proc/self/maps
+    cat /proc/self/maps > /dev/null
```

Comment:
What is `/proc/self/maps` and why are ls'ing and piping it to `/dev/null`? I think a short comment would for future reference here :)

Reply:
oh wait, that's some debugging stuff I need to drop. it was there just to confirm /proc/self is there and